packages/shared-ux/chrome/serverless_project_navigation.mdx

```diff
@@ -473,40 +353,15 @@ import { type NavigationTreeDefinition, getPresets } from '@kbn/shared-ux-chrome
 const navigationTree: NavigationTreeDefinition = {
   body: [
     {
-      type: 'navGroup',
-      ...getPresets('ml'), // Insert all the machine learning links
+      type: 'preset',
+      preset: 'ml', // Insert all the machine learning links
     },
   ],
 };
 ```
 
 The benefit of using the preset instead of building the platform sections manually is that the team who owns the platform area can update the preset as they see fit.
 
-You can also customize the preset like so:
```

Comment:
`getPresets` is not exported from the Chrome package. We can add it if there is a need for it in the future. For now I try to limit the API surface